lib/std/fs/file.zig

```diff
@@ -170,7 +170,13 @@ pub const File = struct {
     pub fn isTty(self: File) bool {
         return os.isatty(self.handle);
     }
-
+    pub fn isDumbTerm(self: File) bool {
```

Comment:
TFW `File.isDumbTerm` doesn't depend on the file parameter at all.
Isn't this `supportsAnsiEscapeCodes` (but wronger)?

Shitty API anyway, this has nothing to do with files and everything to do with the execution environment.